.git-blame-ignore-revs

```diff
@@ -1,2 +1,3 @@
 # Whitespace-only changes
 d01ba56c2127789d85723793380a7378394583f1
+852c59e89088455d251376786776fd556784995c
```

Comment:
This is https://github.com/oxidecomputer/omicron/commit/852c59e89088455d251376786776fd556784995c which we don't want to affect blame with.